### PR TITLE
Add jmol

### PR DESCRIPTION
**Pull request by saraedum**

### CODE REVIEW

**Comment by saraedum** [comment]
@isuruf: This is part of https://github.com/conda-forge/sagelib-feedstock/issues/19 and https://github.com/conda-forge/sage-feedstock/issues/2

**Comment by saraedum** [comment]
I tried this locally, and `jmol` launched the `Jmol` GUI with this recipe.

File: recipes/jmol/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/jmol/files/Jmol/Version {{ version }}/Jmol {{ version }}.{{ patchlevel }}/Jmol-{{ version }}.{{ patchlevel }}-binary.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
```
Comment (saraedum) [inline recipes/jmol/meta.yaml]: Is there another `noarch:` value? This isn't Python of course…

Comment (isuruf) [reply recipes/jmol/meta.yaml]: generic

File: recipes/jmol/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{% set name = "Jmol" %}
+{% set version = "14.29" %}
```
[the remark this answers is not in the document]
Comment (isuruf) [reply recipes/jmol/meta.yaml]: https://github.com/conda-forge/tk-feedstock/blob/1ddb62047b5dc0bd31928c38f66f09bca5534cfd/recipe/meta.yaml#L1-L2

File: recipes/jmol/build.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+set -ex
+mkdir -p "$PREFIX/share/jmol"
+unzip jsmol.zip -d "$PREFIX/share/jmol"
```
[the remark this answers is not in the document]
Comment (isuruf) [reply recipes/jmol/build.sh]: Sure. What's the difference?

**Pull request merged**